MonoidalCategories/doc/Doc.autodoc

```diff
@@ -0,0 +1,2 @@
+@Chapter Tensor Product and Internal Hom
```

Comment:
This file is not necessary

Reply:
I would suggest to leave this file, as it will happen that there appear more chapters in the future.

Reply:
I agree.